aquifer/aquifer-git
# Replace nodegit with Aquifer Run API commands

This might be over simplified but it's working in my test instance.

**To test:**
- Run `aquifer deploy-git` 
- Verify your build gets deployed.
